roofit/roofitcore/inc/RooNLLVarNew.h

```diff
@@ -0,0 +1,65 @@
+#ifndef RooNLLVarNew_h
```

Comment:
(c), license (I'll stop commenting that now)

Reply:
Added! For all new files and also the existing ones in `batchcompute` that still didn't have one.